test/server/cards/01_SOR/events/MaximumFirePower.spec.js

```diff
@@ -0,0 +1,76 @@
+describe('Maximum Firepower', function() {
+    integration(function () {
+        describe('Maximum Firepower\'s ability', function () {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['maximum-firepower'],
+                        groundArena: ['death-trooper'],
+                        spaceArena: ['tieln-fighter', 'gladiator-star-destroyer'],
+                    },
+                    player2: {
+                        groundArena: ['wampa']
```

Comment:
Can you add an enemy Imperial unit in both `describe` blocks so that we can confirm it doesn't do something weird like try to attack with an opponent's unit

Reply:
Absolutely!